QuizApp/QuizSession/ViewControllers/QuizSessionViewController.swift

```diff
@@ -137,6 +145,23 @@ extension QuizSessionViewController: ConstructViewsProtocol {
 // MARK: ProgressBar functions
 extension QuizSessionViewController {
 
+    func bindViews() {
+        questionView
+            .$isCorrectAnswer
+            .sink {  [weak self] isCorrect in
+                guard let self = self, let isCorrect = isCorrect else { return }
+
+                self.colorProgressViews(isCorrect: isCorrect)
+                self.nextQuestion()
+            }
+            .store(in: &cancellables)
```

Comment:
what happens when the user answered incorrectly?

Reply:
If the user answers the question incorrectly then in the ` ` ` colorProgressViews()` ` `  function colors the top progress view that represents that question red.